clients.json

```diff
@@ -1875,5 +1875,15 @@
     "description": "A custom connector for Dell Boomi that utilizes the lettuce.io Java client to add Redis client support to the Dell Boomi iPaaS.",
     "authors": [],
     "active": true
+  },
+
+  {
+    "name": "Redis library for SWI-Prolog",
+    "language": "Prolog",
+    "repository": "https://github.com/SWI-Prolog/packages-redis",
+    "description": "Prolog redis client that exploits SWI-Prolog's extensions such as strings for compact replies and threads to deal with publish/subscribe.",
+    "authors": ["Jan Wielemaker", "Sean Charles"],
```

Comment:
These should be Twitter handles if you use it. If not, just remove the key.

Reply:
Thanks for picking this up.   I had forgotten about this.   Would still be happy to have an official pointer.   I don't understand your remark.   I'm not into Twitter.  Please handle the PR as you seem fit.  

I should update the README.  The current library is far more mature than what is suggested there.

Reply:
```suggestion
```